test/widgets/emoji_reaction_test.dart

```diff
@@ -350,6 +350,8 @@ void main() {
     }
 
     final searchFieldFinder = find.widgetWithText(TextField, 'Search emoji');
+    Finder findInPicker(Finder finder) =>
```

Comment:
I like the name of this.

It should have a blank line to separate from the other declaration, because they're not both one-line declarations (nor closely related):
```suggestion
    final searchFieldFinder = find.widgetWithText(TextField, 'Search emoji');

    Finder findInPicker(Finder finder) =>
```